code-challenges.test.js

```diff
@@ -15,40 +15,100 @@
+//
+// })
+//
+// // Red this fail. I change from writting my array to just the name of the cariable but it still gave me the same.
+//
+// // b) Create the function that makes the test pass.
+ // //My fisrt step is to declare my new variable.
+// //I am going to try to use a foor loop the things that i want to do is remove my first index and next suffle the rest./ //Using .shift im going to remove the zero index
+ // const removeAndSuffle = (array) => {
+ //      array === array.shift()
+ //    for ( let i= array.length -1; i > 0; i--){
+ //            let randomArray = Math.random();
+ //   }
+ //    return array
+ // }
+
+
+// I spend hours stuck first because i had an error on my expect a missing () and it took me alot to create my for loop I still have trouble with for loops.
+// In my forloop im going to  declare my removeAndSuffle variable this = to an array and i want my array to remove the first index so im going to use .shift.
+//In my forloop I have to say that my length minus 1, and after that i need to use Math.random to shuffle.
+// Its Greennnnnnnnnn!
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
-// Expected output: [5, 109]
-
 
+// Expected output: [-8, 90]
 
-// b) Create the function that makes the test pass.
+// Expected output: [5, 109]
+// First im going to describe and here im going to write what i want.
+//In my "it" im going to write what i want in return i want a new array with just the smallest num and the greatest from the array given.
+// In my expect i want my function to use the variables given anfd in return i want them to just give me two digits.
+// describe( "smallestAndGreates", () => {
+//   it("returns an array of the minimum and maximum numbers in that order", () => {
+//    expect(smallestAndGreates(nums1)).toEqual([-8, 90]);
+//    expect(smallestAndGreates(nums2)).toEqual([5, 109])
+//   })
+//   var nums1 = [3, 56, 90, -8, 0, 23, 6]
+//   var nums2 = [109, 5, 9, 67, 8, 24]
+// })
+// //Its red.
+// // b) Create the function that makes the test pass.
+// //Im going to declare my variable smallestAndGreates equal to array and make a new array.
+// // Im going to try a for loop and im going to use something new that i dind on MDN and is Math.max and Math.min to get the bigest ans smallest num.
+//
+//  const smallestAndGreates = (array) => {
+//    let newArr = []
+//    for(let i= array.length -1;i--){
+//      newArr.push(Math.max(...array))
+//      newArr.push(Math.min(...array))
+//    }
+//
+// return newArr.sort((a,b) => a-b)
+//  }
+
+//Im stuck my test is telling me that i do get my answer but myltiple times and i dont know how to fix this HELPPPPPPPPPP!!!
```

Comment:
I think you are super close. It feels like you have all the right pieces. It may be that you just have a little more than you need. I think the Math.min and Math.max might be enough.